src/components/structures/TimelinePanel.js

```diff
@@ -1021,7 +1021,7 @@ var TimelinePanel = React.createClass({
             if ((allowPartial && boundingRect.top < wrapperRect.bottom) ||
                 (!allowPartial && boundingRect.bottom < wrapperRect.bottom)) {
                 let latestReadEventIndex = i;
-                // Place the RM at a hidden event below the latest seen event (if exists)
+                // Place the RM at a hidden event below the latest visible event.
                 // to prevent RM going up the timeline between clients which do not hide the same events.
```

Comment:
The wording still seems confusing. How about something like "if the RM is on an event that is not visible, find the first event that is visible and place the RM on the event before it."